Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.8.6-buster as base
+
+ENV PYTHONFAULTHANDLER=1 \
+	PYTHONUNBUFFERED=1 \
+	PYTHONHASHSEED=random \
+	PIP_NO_CACHE_DIR=off \
+	PIP_DISABLE_PIP_VERSION_CHECK=on \
+	PIP_DEFAULT_TIMEOUT=100 \
+	POETRY_VERSION=1.1.0
+RUN pip install "poetry==$POETRY_VERSION"
+
+COPY . ./app
```

Comment:
Also, you are doing this for both development and production. Really you should only be doing it for production. Your README should have instructions to run the development container with a mounted volume so that it can take advantage of hot reloading - i.e. you should be able to modify a file and see the change reflected in the app running in docker without rebuilding anything

Even without the mounted volume requirement, you should be updating your README with instructions for running the app in Docker.